### ISSUE TITLE
Setting GridView.ItemTemplate from XamlReader does not work

### ISSUE THREAD

**Issue by double28** [unoplatform/uno#183]
## I'm submitting a...

 Bug report (I searched for similar issues and did not find one)  
 

## Current behavior

I define a GridView in my xaml file,and i can define GridView.ItemsPanel like this:
```` xml
 <GridView.ItemsPanel>
            <ItemsPanelTemplate>
                <ItemsWrapGrid   Orientation="Horizontal" ItemWidth="{Binding ItemWidth}" />
            </ItemsPanelTemplate>
        </GridView.ItemsPanel>
````
but, in my application,I'd like to change the ItemsPanle,so I write some code in my cs file,like this:
```` csharp
string template = "<ItemsPanelTemplate xmlns=\"http://schemas.microsoft.com/winfx/2006/xaml/presentation\"><ItemsWrapGrid Orientation=\"Horizontal\" ItemWidth=\"{0}\" /></ItemsPanelTemplate>";
_gv.ItemsPanel = (ItemsPanelTemplate)XamlReader.Load(string.Format(template, itemWidth));
````

## Expected behavior

I expecte GridView use the ItemsPanel defined by code,but not a blankPage.
There is no business with the ItemWidth property.

Related to https://github.com/unoplatform/uno/issues/183

**Comment by jeromelaban** [comment]
Thanks for the report!

As @davidjohnoliver would probably mention, `ListViewBase` inheriting controls are a bit special, and the scenario for updating the `ItemsPanel` this way is not yet supported. That being said, you could probably try doint it this way:

```csharp
#if HAS_UNO
_gv.ItemsPanel = new ItemsPanelTemplate(() => new ItemsWrapGrid { ItemWidth=42 });
#else
string template = "<ItemsPanelTemplate xmlns=\"http://schemas.microsoft.com/winfx/2006/xaml/presentation\"><ItemsWrapGrid Orientation=\"Horizontal\" ItemWidth=\"{0}\" /></ItemsPanelTemplate>";
_gv.ItemsPanel = (ItemsPanelTemplate)XamlReader.Load(string.Format(template, itemWidth));
#endif
```

You can find an example of the XAML code generated by looking at the `obj/../g/XamlGenerator` folder, and use a similar code.

**Comment by jeromelaban** [comment]
This is now fix in Uno 3.0-dev.
